src/test/scala/TileSpec.scala

```diff
@@ -0,0 +1,11 @@
+import org.scalatest._
+
+class TileSpec extends FlatSpec with Matchers {
+  it should "return correct tile letter value" in {
+    Tile('A').value should be(1)
+    Tile('Z').value should be(10)
+    Tile('C').value should be(3)
+    Tile('G').value should be(2)
+    Tile('K').value should be(5)
+  }
+}
```

Comment:
you can use some code coverage runners in IntelliJ  and that will help you understand which code is not covered if any.  See the run with coverage in the menu.
![image](https://user-images.githubusercontent.com/7304387/51268036-f9f78000-19b6-11e9-8b29-65406bb3a42a.png)


They also can run in your CI build. I have used https://coveralls.io which is relatively easy to set up with https://travis-ci.org
This might all be

Reply:
Might be a good candidate for using scala test generators as well.